include/openssl/ec.h

```diff
@@ -460,6 +460,9 @@ OPENSSL_EXPORT OPENSSL_DEPRECATED unsigned char *EC_GROUP_get0_seed(
 OPENSSL_EXPORT OPENSSL_DEPRECATED size_t
 EC_GROUP_get_seed_len(const EC_GROUP *group);
 
+// ECPKParameters_print returns 1.
```

Comment:
nit: you might want to call out that this doesn't actually print anything.

Reply:
No problem, I was following the pattern we had for other no-ops. But the pattern is a bit too simplistic to be honest.